src/thor-client.ts

```diff
@@ -507,8 +508,58 @@ class ThorClient {
 
 const testURL = process.env.TEST_URL || 'http://localhost:8669'
 
+class LoadBalancedClient {
+    private readonly clients: ThorClient[]
+    private readonly sdkClients: _ThorClient[]
+
+    constructor(urls: string[]) {
+        this.clients = urls.map((url) => new ThorClient(url))
+        this.sdkClients = urls.map(
+            (url) => new _ThorClient(new HttpClient(url)),
+        )
+    }
+
+    get raw(): ThorClient {
+        const handler = {
+            get: (target: ThorClient, prop: keyof ThorClient) => {
+                const client = this.getRandomClient()
+                const value = client[prop]
+                return typeof value === 'function' ? value.bind(client) : value
+            },
+        }
+
+        return new Proxy(this.getRandomClient(), handler)
+    }
+
+    get sdk(): _ThorClient {
+        const handler = {
+            get: (target: _ThorClient, prop: keyof _ThorClient) => {
+                const client = this.getRandomSDKClient()
+                const value = client[prop]
+                return typeof value === 'function' ? value.bind(client) : value
+            },
+        }
+
+        return new Proxy(this.getRandomSDKClient(), handler)
+    }
+
+    private getRandomClient(): ThorClient {
+        return this.clients[Math.floor(Math.random() * this.clients.length)]
+    }
+
+    private getRandomSDKClient(): _ThorClient {
+        return this.sdkClients[
+            Math.floor(Math.random() * this.sdkClients.length)
+        ]
+    }
+}
+
+const Client = new LoadBalancedClient(testEnv.urls)
```

Comment:
This should be the only export, we don't need to export `Node1Client` and `SDKClient` below